bin/rucio-storage-consistency-actions

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Authors:
+# - Stefan Piperov <stefan.piperov@cern.ch>, 2021
+
+"""
+Storage-Consistency-Actions is a daemon to perform the
+deletion of dark files and re-transfer missing files at
```

Comment:
I'd say "trigger for deletion and re-transfer" since that's more accurate

Reply:
Agree - will change.